src/lib/message/CHIPMessageLayer.cpp

```diff
@@ -146,8 +146,12 @@ CHIP_ERROR ChipMessageLayer::Init(InitContext * context)
     OnUnsecuredConnectionCallbacksRemoved = NULL;
     OnAcceptError                         = NULL;
     OnMessageLayerActivityChange          = NULL;
+#if CHIP_CONFIG_MAX_CONNECTIONS
     memset(mConPool, 0, sizeof(mConPool));
+#endif
+#if CHIP_CONFIG_MAX_TUNNELS
```

Comment:
I don't think tunnels are part of CHIP , can we remove these and the #defines?
https://github.com/project-chip/connectedhomeip/pull/2105